packages/bot-web-ui/src/pages/tutorials/faq-content/__tests__/faq-content.spec.tsx

```diff
@@ -48,18 +26,78 @@ describe('<FAQContent />', () => {
                 </DBotStoreProvider>
             </StoreProvider>
         );
-        render(<FAQContent faq_list={mockedFaqContent} />, {
-            wrapper,
-        });
     });
 
     it('should render FAQContent', () => {
-        expect(screen.getByTestId('id-faq__wrapper')).toBeInTheDocument();
+        render(<FAQContent faq_list={faq_content} />, { wrapper });
+        expect(screen.getByTestId('dt_faq_wrapper')).toBeInTheDocument();
+    });
+
+    it('should show content after clicking on accordion', () => {
+        render(<FAQContent faq_list={faq_content} />, { wrapper });
+        const accordion = screen.getByTestId('dt_accordion_test');
+        userEvent.click(accordion);
+        expect(accordion).toHaveTextContent('Deriv Bot is a web-based strategy builder for trading digital options.');
+    });
+
+    it('should show faq content in text size-s on desktop', () => {
+        mock_store.ui.is_desktop = true;
+
+        render(<FAQContent faq_list={faq_content} />, { wrapper });
+        const faq = screen.getByText('What is Deriv Bot?');
```

Comment:
@farabi-deriv let's create a variable for this text as it is being used in multiple places.

Reply:
Sure @sandeep-deriv